processing/src/main/java/io/druid/query/groupby/orderby/OrderByColumnSpec.java

```diff
@@ -202,7 +202,7 @@ private static StringComparator determinDimensionComparator(Object dimensionOrde
     }
 
     String dimensionOrderString = dimensionOrderObj.toString().toLowerCase();
-    return StringComparators.makeComparator(dimensionOrderString);
+    return StringComparator.fromString(dimensionOrderString);
```

Comment:
@fjy I think that'll require some more changes to the way OrderByColumnSpec is constructed, can we handle this in another PR?